salt/modules/yumpkg.py

```diff
@@ -1853,6 +1856,27 @@ def upgrade(name=None,
 
         .. versionadded:: 2016.3.0
 
+    minimal : False
+        Use upgrade-minimal instead of upgrade (e.g., ``yum upgrade-minimal``)
+        Goes to the 'newest' package match which fixes a problem that affects your system.
+
+        .. code-block:: bash
+
+            salt '*' pkg.upgrade minimal=True
+
+        .. versionadded:: Fluorine
+
+    obsoletes : True
+        Controls wether dnf/yum should take obsoletes into account and remove them.
```

Comment:
Could you make this say "yum/dnf" instead? Just to be consistent with other similar references elsewhere in the docstrings.